FunctionApp/Function1/Function1.cs

```diff
@@ -1,31 +1,32 @@
 ﻿using System.Collections.Generic;
 using System.Net;
+using System.Reflection.PortableExecutable;
 using System.Text.Json;
 using Microsoft.Azure.Functions.Worker;
-using Microsoft.Azure.WebJobs;
-using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
+using Microsoft.Azure.Functions.Worker.Extensions.Http;
+using Microsoft.Azure.Functions.Worker.Extensions.Storage;
+using Microsoft.Azure.Functions.Worker.Http;
```

Comment:
Should this just be `Microsoft.Azure.Functions.Worker` and `Microsoft.Azure.Functions.Worker.Http`?

Reply:
Same comment for all below!